ramalama/console.py

```diff
@@ -0,0 +1,39 @@
+import os
+import logging
+
+
+def is_locale_utf8():
+    """Check if the system locale is UTF-8."""
+    return 'UTF-8' in os.getenv('LC_CTYPE', '') or 'UTF-8' in os.getenv('LANG', '')
+
+
+def supports_emoji():
+    """Detect if the terminal supports emoji output."""
+    term = os.getenv("TERM")
+    if not term or term in ("dumb", "linux"):
+        return False
+
+    return is_locale_utf8()
+
+
+# Allow users to override emoji support via an environment variable
+# If RAMALAMA_FORCE_EMOJI is not set, it defaults to checking supports_emoji()
+RAMALAMA_FORCE_EMOJI = os.getenv("RAMALAMA_FORCE_EMOJI")
+FORCE_EMOJI = RAMALAMA_FORCE_EMOJI.lower() == "true" if RAMALAMA_FORCE_EMOJI else None
+EMOJI = FORCE_EMOJI if FORCE_EMOJI is not None else supports_emoji()
+
+
+# Define emoji-aware logging messages
+def error(msg):
```

Comment:
The spacing isn't consistent in the printed output in these three functions, some are double-space some are single-space.

If you are pushing new commits you might as well catch, we can also do in a follow on PR.